src/libpriv/rpmostree-origin.cxx

```diff
@@ -809,6 +809,8 @@ rpmostree_origin_remove_packages (RpmOstreeOrigin  *origin,
                                   gboolean         *out_changed,
                                   GError          **error)
 {
+  if (!packages)
+    return TRUE;
```

Comment:
This bit is missing for `rpmostree_origin_add_packages` for consistency. Though meh.. kinda leaning towards not even having it since the code already handles it fine. But obviously fine as is.

Reply:
True, will look at this in a followup!